starlark/testdata/builtins.star

```diff
@@ -18,6 +18,24 @@ none = None
 _1 = none and none[0]      # rhs is not evaluated
 _2 = (not none) or none[0] # rhs is not evaluated
 
+# abs
+assert.eq(abs(2.0), 2.0)
+assert.eq(abs(0.0), 0.0)
+assert.eq(abs(-2.0), 2.0)
+assert.eq(abs(2), 2)
+assert.eq(abs(0), 0)
+assert.eq(abs(-2), 2)
+assert.eq(abs(float("inf")), float("inf"))
+assert.eq(abs(float("-inf")), float("inf"))
+assert.eq(abs(float("nan")), float("nan"))
+assert.fails(lambda: abs("0"), "must be real number, not string")
+def abs_bigint():
```

Comment:
I assume you introduce the def here to avoid defining maxint32 at top-level, but it's ok. Let's inline the call.